src/main/java/org/opentripplanner/middleware/utils/DateTimeUtils.java

```diff
@@ -154,9 +154,6 @@ public static ZoneId getSystemZoneId() {
      */
     public static ZoneId getOtpZoneId() {
         String otpTzId = getConfigPropertyAsText("OTP_TIMEZONE");
-        if (otpTzId == null) {
-            throw new RuntimeException("OTP_TIMEZONE is not defined in config!");
-        }
```

Comment:
Might be good to check where the required args are referenced and remove these similar checks. For example, `BugsnagDispatcher#makeEventDataRequest`.

Reply:
@landonreed I can't see what needs to be updated in `BugsnagDispatcher#makeEventDataRequest` which is a concern as I'll probably miss other similar updates (I have updated more obvious required args references in other classes/methods).

Reply:
Oh, I think it disappeared with the merge of dev. I think we're probably ok on this front.